channels.yaml

```diff
@@ -223,14 +223,14 @@ releases:
     featureVersions: *featureVersions-v1
   - version: v1.22.13+k3s1
     minChannelServerVersion: v2.6.3-alpha1
-    maxChannelServerVersion: v2.7.99
+    maxChannelServerVersion: v2.6.99
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
   # v1.22.14+k3s1 was never released
   - version: v1.22.15+k3s1
     minChannelServerVersion: v2.6.3-alpha1
-    maxChannelServerVersion: v2.7.99
+    maxChannelServerVersion: v2.6.99
```

Comment:
Same for k3s, update default version because v1.22.15 won't show up

Reply:
Please check my reply to another comment: https://github.com/rancher/kontainer-driver-metadata/pull/987#discussion_r996034796